<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>570</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(11, 25, 25);
border-radius: 25px</string>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
background:transparent;
border-radius:100px;</string>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>570</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(35, 39, 39);
border-radius: 25px</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_text">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>181</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(105, 118, 132, 255);
color:rgba(255, 255, 255, 230);
padding-bottom:7px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_accept">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>240</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_accept
{
background-color:rgb(136, 147, 147);
border-radius:5px;
font-size:15px
}
QPushButton#btn_accept:hover
{
background-color: rgb(255, 255, 255);
}
QPushButton#btn_accept:pressed
{
font-size:13px
}</string>
     </property>
     <property name="text">
      <string>Далее</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_close">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>240</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_close
{
background-color:rgb(136, 147, 147);
border-radius:5px;
font-size:15px
}
QPushButton#btn_close:hover
{
background-color: rgb(255, 255, 255);
}
QPushButton#btn_close:pressed
{
font-size:13px
}</string>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Введите капчу</string>
     </property>
    </widget>
    <widget class="QFrame" name="image_frame">
     <property name="geometry">
      <rect>
       <x>234</x>
       <y>10</y>
       <width>321</width>
       <height>140</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_getFile">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>190</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_getFile
{
background-color:rgb(136, 147, 147);
border-radius:5px;
font-size:15px
}
QPushButton#btn_getFile:hover
{
background-color: rgb(255, 255, 255);
}
QPushButton#btn_getFile:pressed
{
font-size:13px
}</string>
     </property>
     <property name="text">
      <string>Выбрать файл</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>201</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/check/018f5987fde8a21295e00599a97b-yfk1.png);
border-radius: 25px</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_text_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>190</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(105, 118, 132, 255);
color:rgba(255, 255, 255, 230);
padding-bottom:7px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>Вставьте ссылку</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_getFile</sender>
   <signal>clicked(bool)</signal>
   <receiver>btn_accept</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_accept</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
